[2025-07-07T20:07:33.649+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T20:07:22.520882+00:00 [queued]>
[2025-07-07T20:07:33.657+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T20:07:22.520882+00:00 [queued]>
[2025-07-07T20:07:33.657+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-07T20:07:33.674+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-07 20:07:22.520882+00:00
[2025-07-07T20:07:33.683+0000] {standard_task_runner.py:57} INFO - Started process 2098 to run task
[2025-07-07T20:07:33.685+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_activity_analysis', 'transform_data', 'manual__2025-07-07T20:07:22.520882+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/customer_activity_dag.py', '--cfg-path', '/tmp/tmptlrsy8tl']
[2025-07-07T20:07:33.687+0000] {standard_task_runner.py:85} INFO - Job 88: Subtask transform_data
[2025-07-07T20:07:33.701+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T20:07:33.730+0000] {task_command.py:410} INFO - Running <TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T20:07:22.520882+00:00 [running]> on host 7a17032e36f4
[2025-07-07T20:07:33.945+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ds_const@mail.ru' AIRFLOW_CTX_DAG_OWNER='Daniil' AIRFLOW_CTX_DAG_ID='customer_activity_analysis' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T20:07:22.520882+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T20:07:22.520882+00:00'
[2025-07-07T20:07:33.955+0000] {logging_mixin.py:150} INFO - /opt/***/shared_tmp/extract_manual__2025-07-07T20:07:22.520882+00:00.csv
[2025-07-07T20:07:35.079+0000] {logging_mixin.py:150} INFO - Файл /opt/***/shared_tmp/extract_manual__2025-07-07T20:07:22.520882+00:00.csv успешно удалён
[2025-07-07T20:07:35.085+0000] {logging_mixin.py:150} INFO - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!2025-07-07T20:07:22.520882+00:00
[2025-07-07T20:07:35.085+0000] {logging_mixin.py:150} INFO - Received date type: <class 'datetime.datetime'>, value: 2024-04-01 00:00:00
[2025-07-07T20:07:35.494+0000] {logging_mixin.py:150} WARNING - Generating flags:   0%|          | 0/10 [00:00<?, ?it/s]
[2025-07-07T20:07:35.504+0000] {logging_mixin.py:150} WARNING - Generating flags: 100%|##########| 10/10 [00:00<00:00, 1021.43it/s]
[2025-07-07T20:07:35.943+0000] {python.py:183} INFO - Done. Returned value was: /opt/***/shared_tmp/transform_manual__2025-07-07T20:07:22.520882+00:00.csv
[2025-07-07T20:07:35.966+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=customer_activity_analysis, task_id=transform_data, execution_date=20250707T200722, start_date=20250707T200733, end_date=20250707T200735
[2025-07-07T20:07:36.022+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-07-07T20:07:36.056+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
